Lib/python/pyrun.swg

```diff
@@ -11,8 +11,8 @@
 # error "This version of SWIG only supports Python >= 2.7"
 #endif
 
-#if PY_VERSION_HEX >= 0x03000000 && PY_VERSION_HEX < 0x03030000
-# error "This version of SWIG only supports Python 3 >= 3.3"
+#if PY_VERSION_HEX >= 0x03000000 && PY_VERSION_HEX < 0x03050000
+# error "This version of SWIG only supports Python 3 >= 3.5"
 #endif
```

Comment:
This seems to update the hard requirement to Python 3 >= 3.5 but you only updated the docs to say that for the stable ABI case.

Is updating the hard requirement what you intended?  If so, the docs should be updated as currently they say:

> SWIG is compatible with all recent Python versions (Python 2.7 and Python &gt;= 3.5).  Python 3.3 and 3.4 may still work but we are no longer regularly testing with them. SWIG-4.0 supported Python 3.2.  SWIG-3.0 supported older Python 2.x and 3.x.

So delete the second sentence and record the last version to support 3.4.

Reply:
ok